src/styles/blocks/header.scss

```diff
@@ -0,0 +1,107 @@
+.header {
+  $root: &;
+
+  border-bottom: 1px solid transparent;
+
+  transition: border-bottom $transition-props,
+    background-color $transition-props;
+
+  &--scroll {
+    border-bottom: 1px solid $color-dark;
```

Comment:
same about border here, you modifier doesnt change width or type of border just color) so just use `border-color`)